### ISSUE TITLE
Unable to arrange buses to the newly added bus schedule 

### ISSUE THREAD

**Issue by shaomingli96** [jeng6529/Bus-Ticketing-System#2]
the system cannot arrange specific buses to the newly added bus schedule may cause the buses non-arrival .

**Comment by MichaelLok** [comment]
you mean can't add the new bus to the bus schedule?

**Comment by shaomingli96** [comment]
No, the bus schedule should assign bus id to fix the position of a bus so that it wont happen the buses non arrival

**Comment by chongyimiao** [comment]
is the bus id same with the license plate?

**Comment by taiyungtan96** [comment]
no..this bus id is different with the the licence plate number. However, the bus id is a auto generated number combined with a character 'B' and a series of number. For example, B1012.

**Comment by chongyimiao** [comment]
Oh I see, thankyou.

**Comment by taiyungtan96** [comment]
you are welcome.
